atlasdb-perf/src/main/java/com/palantir/atlasdb/performance/test/generator/RandomValueGenerator.java

```diff
@@ -0,0 +1,44 @@
+/**
+ * Copyright 2015 Palantir Technologies
+ *
+ * Licensed under the BSD-3 License (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://opensource.org/licenses/BSD-3-Clause
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.palantir.atlasdb.performance.test.generator;
+
+import java.nio.ByteBuffer;
+import java.util.Random;
+import java.util.stream.Stream;
+
+import com.palantir.atlasdb.performance.test.api.ValueGenerator;
+
+public class RandomValueGenerator implements ValueGenerator {
+
+    private static final int BYTE_ARRAY_SIZE = 100;
+
+    private final Random rand;
+
+    public RandomValueGenerator(Random rand) {
+        this.rand = rand;
+    }
+
+    @Override
+    public Stream<ByteBuffer> stream() {
+        return Stream.generate(this::genBytes);
+    }
+
+    private ByteBuffer genBytes() {
+        byte[] b = new byte[BYTE_ARRAY_SIZE];
```

Comment:
Yes, definitely.  This is currently just filler code to get us off the ground (so we can start parallelizing work -- when writing this code I talked to clocks about your comment).  Once this merges we should be able to work on writing tests and making infrastructure at the same time.

Overall there is a question how "value generators" should work in general.  I think the api (and necessary functionality) will be clear once we write a few simple tests.  I was aiming to make this pluggable so that any tests can be run with different types of value generation, but maybe this is too much abstraction and tests should deal with their own load generation.